gds_write/def2stream.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+# BSD 3-Clause License
+#
+# Copyright (c) 2018-2023, The Regents of the University of California
```

Comment:
is that the original license from which you created the derivative? There might be some implication since bazel_rules_hdl is Apache 2.0, see: https://github.com/hdl/bazel_rules_hdl/blob/main/LICENSE

@QuantamHD any insights?

Reply:
This license is taken from [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/LICENSE_BUILD_RUN_SCRIPTS) but it was never a part of the [source file](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/util/def2stream.py) that I imported. However, the License File states that it applies to all build and run scripts from OpenROAD-flow-scripts so it applies also to `def2stream.py`.

As far as I know, it is possible to import BSD 3-licensed source files into Apache 2.0 project but it is required to keep the original license header in the imported file.